frontend/src/pages/Auth/SignIn.tsx

```diff
@@ -114,6 +120,11 @@ export const SignIn: React.FC<Props> = ({ setResolver }) => {
 	}
 
 	useEffect(() => analytics.page(), [])
+	useEffect(() => {
+		if (!loadingWorkspaceForInvite) {
+			setLoadingState(AppLoadingState.LOADED)
+		}
+	}, [loadingWorkspaceForInvite, setLoadingState])
```

Comment:
We needed to call `setLoadingState(AppLoadingState.LOADED)` here when redirecting from the vercel integration page. This fixes the infinite loading state.